.github/workflows/tests.yaml

```diff
@@ -23,8 +22,10 @@ jobs:
             dependencies: "lowest"
           - php-version: "8.1"
             dependencies: "highest"
+            tests: --fail-on-risky
```

Comment:
A risky test should not be automatically a failing test (and I'm not alone with that [option](https://github.com/sebastianbergmann/phpunit/issues/1612)). 
Anyway I will approve it because the rest of this PR is fine.

Reply:
We already had this behavior before. However, it does not work for the “lowest” tests, as there are deprecations there. I have therefore only deactivated it there for the moment.